packages/toolpad-app/src/toolpadDataSources/rest/server.ts

```diff
@@ -137,20 +137,22 @@ async function execBase(
   connection: Maybe<RestConnectionParams>,
   fetchQuery: FetchQuery,
   params: Record<string, string>,
```

Comment:
I guess that we need to update the type of the `params` argument now: besides having binding values directly, it can now also have bindings (type `BindableAttrValue`)